src/2014.ipynb

```diff
@@ -177,32 +178,6 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "technology ANOMALY: 9 Project Title: Development of a Low-cost, Portable\n",
-      "Photoacoustic Apparatus for Solid Samples\n",
-      "Using High Power Light Emitting Diodes.\n",
-      "Name: Fiachra Kenny\n",
-      "Email: fiachra.kenny5@mail.dcu.ie\n",
-      "Programme: Electronic Engineering\n",
-      "Supervisor: Prof Patrick McNally\n",
-      "Photoacoustic spectroscopy is a method of examining gases liquids and solids by measuring\n",
-      "a photoacoustic signal which is produced by the absorption of light in the sample.\n",
-      "The goal of this project was to develop, build and test a portable photoacoustic setup for solid\n",
-      "samples using high power LEDs. The aim in the development of this setup is to simplify a\n",
-      "conventional photoacoustic setup, reduce the cost and increase its portability.\n",
-      "A large factor in achieving these goals is the use of high power LEDs as a light source instead of a\n",
-      "conventional source such as a laser or Xenon lamp.\n",
-      "The photoacoustic setup developed includes a photoacoustic cell where samples are housed along\n",
-      "with a LED light source and an electret microphone to acquire the photoacoustic signal.\n",
-      "This project outlines all the steps involved in the design, fabrication and assembly of the\n",
-      "photoacoustic chamber as well as the design and use of the signal acquisition electronics and\n",
-      "software. It also details the setup and procedure to run photoacoustic measurement experiments on\n",
-      "solid samples.\n",
-      "Using the developed setup photoacoustic measurements and quantitative analysis of Carbon black,\n",
-      "Silicon and Graphite was performed using a number of LEDs with wavelengths ranging from\n",
-      "395nm-875nm.\n",
-      "Project Area: Photoacoustics\n",
-      "Project Technology: High Power LEDs\n",
-      "\n",
       "platform ANOMALY: 9 Project Title: Development of a Low-cost, Portable\n",
```

Comment:
We can ignore these platform anomalies. It just means that platform will be null since it's not provided.